v22.1/ui-statements-page.md

```diff
@@ -25,7 +25,12 @@ By default, the Statements page shows SQL statements from all applications and d
 
 ### Search statements
 
-To search using the search field, type a string over `Search Statements` and press `Enter`. The list of statements is filtered by the string.
+To search using the search field:
+
+1. Type a string over `Search Statements`. {% include_cached new-in.html version="v22.1.4" %} To search for exact terms in order, wrap the search string in quotes.
+1. Press `Enter`.
```

Comment:
nit: is this suppose to be `2.`?

Reply:
It renders as 2. in the HTML. We number all ordered list items with 1. in case we want to add or remove items.